runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesPlugin.java

```diff
@@ -111,6 +111,11 @@ public void onStatChanged(StatChanged statChanged)
 			return;
 		}
 
+		if (config.showVirtualLevel() && currentLevel >= Experience.MAX_REAL_LEVEL)
```

Comment:
This can be combined with the check above as follows:

```java
if (currentLevel >= Experience.MAX_REAL_LEVEL)
{
	if (config.hideMaxed())
	{
		return;
	}
	else if (config.showVirtualLevel())
	{
		currentLevel = Experience.getLevelForXp(currentXp);
	}
}
```

Reply:
done